Makefile

```diff
@@ -74,8 +74,8 @@ OCI_RUNTIME ?= $(shell which podman || which docker)
 ENVTEST_ASSETS_DIR=$(CURDIR)/testbin
 test: generate go-generate lint manifests
 test: $(GO_FILES)
-	mkdir -p ${ENVTEST_ASSETS_DIR}
-	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.7.0/hack/setup-envtest.sh
+	@mkdir -p ${ENVTEST_ASSETS_DIR}
+	@test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.7.0/hack/setup-envtest.sh
```

Comment:
These are verbose when running `make test` and it's unnecessary output.

Reply:
Do we need the `envtest` at all? We don't really use it right now. Why not dropping this?